apps/test/unit/templates/progress/ProgressLessonTeacherInfoTest.js

```diff
@@ -66,10 +68,9 @@ describe('ProgressLessonTeacherInfo', () => {
         lockableAuthorized={false}
       />
     );
-    assert.equal(
-      wrapper.find('SendLesson').props().lessonUrl,
-      'code.org?login_required=true'
-    );
+    const lessonUrl = wrapper.find('SendLesson').props().lessonUrl;
+    assert(lessonUrl.includes('code.org?login_required=true'));
```

Comment:
Do we want to change this assert (or add a new one) to check that the protocol is part of lessonUrl?

Reply:
Do you have an idea of how to check that? The protocol will be different if we're running on test vs locally, right? I was thinking about stubbing window.location but that seemed like more trouble than it was worth..

Reply:
Oh, hmmm.  Would something like `includes('http')` work?